changelog.d/16251.bugfix

```diff
@@ -0,0 +1 @@
+Fix a longstanding bug where appservices using MSC2049 to receive to_device messages, would only get messages for one user.
```

Comment:
MSC2049 doesn't seem to be a thing?

```suggestion
Fix a long-standing bug where appservices using MSC2049 to receive `to_device` messages, would only get messages for one user.
```

Reply:
The correct MSC is [2409](https://github.com/Sorunome/matrix-doc/blob/soru%2Bhs/appservice-edus/proposals/2409-appservice-edus.md).